zeroconf/_core.py

```diff
@@ -198,12 +198,16 @@ def datagram_received(
         self, data: bytes, addrs: Union[Tuple[str, int], Tuple[str, int, int, int]]
     ) -> None:
         assert self.transport is not None
+        _v6: Union[Tuple[()], Tuple[int, int]] = ()
         if len(addrs) == 2:
             # https://github.com/python/mypy/issues/1178
             addr, port = addrs  # type: ignore
+            _scope = None
         elif len(addrs) == 4:
             # https://github.com/python/mypy/issues/1178
             addr, port, _flow, _scope = addrs  # type: ignore
+            log.debug('IPv6 scope_id %d associated to the receiving interface', _scope)
+            _v6 = (_flow, _scope)
```

Comment:
```suggestion
            addr, port, flow, scope = addrs  # type: ignore
            log.debug('IPv6 scope_id %d associated to the receiving interface', _scope)
            _v6 = (flow, scope)
```

These were only prefixed with underscores to indicate they were unused